pyVMC.py

```diff
@@ -4684,6 +4692,8 @@ def __init__(self,prog):
                                     "Show the SDDC route table:\n"
                                     "python pyMVC.py system show-routes t0 \n \u00A0 \n")
 
+    ap.add_argument('auth_method', choices=['oauth', 'refresh_token'], help = "Choose either 'oauth' or 'refresh_token' to indicate how pyVMC authenticates to the VMware Cloud Service Portal.  Be sure the correct parameters are configured in config.ini.")
```

Comment:
Can we set a default of refresh_token? I'm not seeing value in forcing the user to specify the auth type every time they run PyVMC.

Reply:
Easy enough - I will make that change.

Reply:
Agree with @kremerpatrick should be a silent default.